DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Queries/PreferencesQuery.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.Preferences.DAL.Model;
+
+namespace GitHubExtention.Preferences.WebApi.Queries
+{
+    class PreferencesQuery : IPreferencesQuery
+    {
+        private readonly PreferencesContext _context;
+
+        public PreferencesQuery(PreferencesContext context)
+        {
+            _context = context;
+        }
+
+        public IOrderedQueryable<User> Users
```

Comment:
I see a small smell...You are exposing Users property and also declare GetUser method in interface? Do not many methods inside interface? Where our flexibility related to IOrderedQueryable?